Makefile

```
@@ -40,7 +40,7 @@ cover: test
 
 # Check the formatting
 .PHONY: fmt
-fmt:
+fmt: 
```

```suggestion
fmt:
```

(minor nit: trailing whitespace)